.registry/resources/minimalaws.ui-schema.yaml

```
@@ -1,4 +1,4 @@
-version: v0.2.0
+version: v0.2.1
 configSections:
 - title: Minimal AWS Configuration
```

I think this section title should be the following:

```suggestion
- title: Provider Details
```